docs/src/extensions.md

```diff
@@ -22,8 +22,12 @@ $ ls -w=80
 
 ## `env`
 
-`env` has an additional `-f`/`--file` flag that can parse `.env` files and set
-variables accordingly. This feature is adopted from `dotenv` style packages.
+GNU `env` allows the empty string to be used as an environment variable name. This is explicitly
+unsupported by the Rust standard library (see the
```

Comment:
The rust part of this seems to be an implementation detail
not sure it is interesting to mention it here, no ?